app.js

```diff
@@ -49,6 +49,7 @@ var app = express();
 // Bootstrap application settings
 app.use(express.static('./public')); // load UI from public folder
 app.use(bodyParser.json());
+require("cf-deployment-tracker-client").track();

```

Comment:
looks correct according to the docs: https://github.com/IBM-Bluemix/cf-deployment-tracker-client-node#to-use